.github/workflows/broken-linkcheck.yml

```
@@ -32,7 +33,31 @@ jobs:
           set -o pipefail
           python make.py linkcheck | tee linkcheck.txt
 
+      - name: Create or Update Linkcheck Issue
+        if: github.event_name == 'pull_request'
```

All steps are skipped following a failed one unless you explicitly tell GHA to run it:

```suggestion
        if: failure() && github.event_name == 'pull_request'
```